week3/homework/src/index.js

```diff
@@ -13,35 +13,34 @@ const editToDo = require('./editToDo');
 app.use(express.json());
 app.use(bodyParser.json());
 
-app.post('/addTodo', (req, res) => {
-  const toWrite = req.body.todo.description;
+app.post('/creatingNewTodo', (req, res) => {
+  const toWrite = req.body.description;
   addOne(req, res, toWrite);
 });
 
-app.get('/todos', (req, res) => {
+app.get('/todo', (req, res) => {
   getList(req, res);
 });
 
-app.get('/todo/?', (req, res) => {
-  const id = req.query.id;
-  getToDo(req, res, id);
+app.get('/todo/:id', (req, res) => {
+  getToDo(req, res, req.params.id);
 });
 
 app.delete('/todos', (req, res) => {
   deleteAll(req, res);
 });
 
-app.put('/todo/?', (req, res) => {
-  deleteOne(req, res, req.query.id);
+app.put('/todo/:id', (req, res) => {
```

Comment:
This should be `app.delete('/todo/:id', (req, res) ...`

Reply:
I thought that it should be post because it is just updating the data..
i fix it 
thanks a lot!